### PR TITLE
Feature/moz 383 add matrix to groups

### PR DESCRIPTION
**Pull request by kycapstick**
Added input for Matrix room to groups creation/edit and added link to Community Links on single group page.

### CODE REVIEW

**Comment by dnaluz** [comment]
@kycapstick great work adding the validation check for the matrix field on the person profile.  Add it here and this should be good to go!

**Comment by kycapstick** [comment]
@dnaluz Updated styles and added check to Matrix ID on group creat/edit

File: js/groups.js [hunk 1]
```diff
@@ -12,9 +12,41 @@ jQuery(function(){
             }
             e.stopPropagation();
         }
-
-        
-    });
+	});
+	
+	const checkMatrixValue = function(value) {
+		const regex = new RegExp(/^[a-z0-9.\-_=/]+:[A-Za-z0-9[]+/, 'gi');
```
Comment (dnaluz) [inline js/groups.js]: Should we be validating that the domain part is indeed a valid domain?

Comment (kycapstick) [reply js/groups.js]: I was going to add this fix, but there's no standard format for domains in Matrix. Any of the formats outlined here could technically be valid:

https://github.com/matrix-org/matrix-doc/blob/master/specification/appendices/identifier_grammar.rst#server-name

Comment (dnaluz) [reply js/groups.js]: They are 3 standard formats IPv4, IPv6 and regular domains.  This should be able to handle them I believe, try some tests it just a javascript regular expression.

`/^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$|^\s*((([0-9A-Fa-f]{1,4}:){7}([0-9A-Fa-f]{1,4}|:))|(([0-9A-Fa-f]{1,4}:){6}(:[0-9A-Fa-f]{1,4}|((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){5}(((:[0-9A-Fa-f]{1,4}){1,2})|:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){4}(((:[0-9A-Fa-f]{1,4}){1,3})|((:[0-9A-Fa-f]{1,4})?:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){3}(((:[0-9A-Fa-f]{1,4}){1,4})|((:[0-9A-Fa-f]{1,4}){0,2}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){2}(((:[0-9A-Fa-f]{1,4}){1,5})|((:[0-9A-Fa-f]{1,4}){0,3}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){1}(((:[0-9A-Fa-f]{1,4}){1,6})|((:[0-9A-Fa-f]{1,4}){0,4}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:))|(:(((:[0-9A-Fa-f]{1,4}){1,7})|((:[0-9A-Fa-f]{1,4}){0,5}:((25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)(\.(25[0-5]|2[0-4]\d|1\d\d|[1-9]?\d)){3}))|:)))(%.+)?\s*$/`

**Pull request merged**